oauth2-example-app/src/main/java/com/clouway/oauth2/exampleapp/LoginEndpoint.java

```diff
@@ -26,7 +30,7 @@ public LoginEndpoint(ResourceOwnerAuthentication authentication) {
   }
 
   @Post
-  public String login(HttpServletRequest request, HttpServletResponse response) {
+  public String login(HttpServletRequest request, HttpServletResponse response) throws UnsupportedEncodingException {
```

Comment:
Would be better if the error is handled and not thrown by the Post method.

Reply:
Okay should I throw an Error instead like this?

 catch(... e) {throw new Error("The world is ending, UTF-8 is not an encoding anymore!", e);}

or maybe I could use URLEncoder.encode without the encoding parameter (but that is deprecated API)